src/browser/modules/App/App.jsx

```diff
@@ -194,12 +194,15 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
   const setInitialConnectionData = (graph, credentials, context) => {
     const creds = getActiveCredentials('bolt', context)
     if (!creds) return // No conection. Ignore and let browser show connection lost msgs.
+    const httpCreds = getActiveCredentials('http', context)
+    const restApi = `http://${httpCreds.host}:${httpCreds.port}`
```

Comment:
Would it be possible to use https if it's available, and http just as a fallback?